pom.xml

```diff
@@ -9,7 +9,7 @@
     <version>1.0-SNAPSHOT</version>
     <properties>
         <java.version>11</java.version>
-        <keycloak.version>11.0.0</keycloak.version>
+        <keycloak.version>12.0.0</keycloak.version>
```

Comment:
Since you're bumping the Keycloak version, could you also update the Compatibility Chart in `README.md`?